cassandra/cluster.py

```diff
@@ -259,6 +268,23 @@ class ProfileManager(object):
     def __init__(self):
         self.profiles = dict()
 
+    def _validate_contact_points(self, contact_points):
```

Comment:
How about this?:

- rename to `_check_for_lbps` or something like that
- remove the `contact_points` param and `if contact_points is _NOT_SET` from this method
- call this as `if contact_points is _NOT_SET: self.profile_manager._check_for_lbps` in `Cluster.__init__`

I'll make a note in `Cluster` where this would change.

Reply:
`_check_for_lbps` or `_validate_lbps` sounds good.